prboom2/src/r_draw.c

```diff
@@ -475,13 +478,13 @@ void R_InitBuffersRes(void)
 
 void R_InitBuffer(int width, int height)
 {
-  int i;
-
   drawvars.topleft = screens[0].data;
   drawvars.pitch = screens[0].pitch;
 
-  for (i=0; i<FUZZTABLE; i++)
-    fuzzoffset[i] = fuzzoffset_org[i]*screens[0].pitch;
+  if (!tallscreen)
+    fuzzcellsize = (SCREENHEIGHT + 100) / 200;
```

Comment:
Is the `+ 100` and `+ 160` here basically just for rounding?

Reply:
Yeah, rounding to nearest